engine/installation/linux/oracle.md

```diff
@@ -94,7 +116,27 @@ Docker from the repository.
     ```bash
     $ sudo yum -y check-update
```

Comment:
This command does not update the `yum` package index. In order to do that, you should run `yum makecache fast` instead.